gluoncv/utils/metrics/coco_keypoints.py

```diff
@@ -128,6 +128,9 @@ def update(self, preds, maxvals, score, imgid, *args, **kwargs):
             if count > 0:
                 kpt_score /= count
             rescore = kpt_score * score[idx].asscalar()
+            fixed_score = kwargs.get('fixed_score', False)
```

Comment:
Can I know what is `fixed_score` and the reason for fixing rescore to 2? Thank you.

Reply:
Good catch, it was for DSNT modules which only outputs the coordinates but no score. Now I fix the score outside the function call so it should be removed. Thanks.